javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,149 @@
+//   to log out the text: Called after 5 seconds 5 seconds after the button is clicked.
+//**************************************************************************************
+const button1 = document.getElementById("ex3");
+
+button1.addEventListener("click", ex3);
+
+function ex3() {
+  ex2(5, "Called after 5 seconds");
+}
+
+//**************************************************************************************
+// 4. Create two functions and assign them to two different variables.
+//    One function logs out Earth, the other function logs out Saturn.
+//    Now create a new third function that has one parameter: planetLogFunction.
+//    The only thing the third function should do is call the provided parameter function.
+//    Try call the third function once with the Earth function and once with the Saturn function.
+//**************************************************************************************
+
+const x = () => {
+  console.log(`Earth`);
+};
+const y = () => {
+  console.log(`Saturn`);
+};
+
+function logoutPlanet(planetLogFunction) {
+  planetLogFunction();
+}
+
+logoutPlanet(x);
+logoutPlanet(y);
+
+//**************************************************************************************
+// 5. Create a button with the text called "Log location".
+//    When this button is clicked the location (latitude, longitude) of the user should be logged out using this browser api
+//**************************************************************************************
+const logbtn = document.getElementById("logcation");
+const userLocation = document.getElementById("yourLocation");
+
+logbtn.addEventListener("click", getLocation);
+
+function getLocation() {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(
+      (position) =>
+        (userLocation.innerHTML =
+          "Latitude : " +
+          position.coords.latitude.toFixed(4) +
+          "<br>Logitude: " +
+          position.coords.longitude.toFixed(4))
+    );
+  } else {
+    logbtn.innerHTML = "Geolocation is not supported by this browser.";
+  }
+}
+
+//**************************************************************************************
+// 6. (Optional) Now show that location on a map using fx the Google maps api
+//**************************************************************************************
+//§§§§§§§§§§§§§§§§§§§§§§§§§§§§§
+// § Skipping Optional §
+//§§§§§§§§§§§§§§§§§§§§§§§§§§§§§
+
+//**************************************************************************************
+// 7. Create a function called runAfterDelay.
+//    It has two parameters: delay and callback.
+//    When called the function should wait delay seconds and then call the provided callback function.
+//    Try and call this function with different delays and different callback functions
+//**************************************************************************************
+
+function runAfterDelay(delay, callback) {
+  let delayTid = delay * 1000;
```

Comment:
Not a bug, but.... 
I believe the naming for variables and functions have to be in English :)
For better understanding your code across developers from around the world ;)